string  msg_name
string  msg_type_hash
uint8   priority
uint8[] msg_content
---
string new_hash
